sdk/include/reactos/debug.h

```diff
@@ -30,6 +30,8 @@
 extern "C" {
 #endif
 
+#ifndef NTOS_MODE_USER
```

Comment:
Actually it would be nice to keep using the original DbgPrint and RtlAssert in case `NTOS_MODE_USER` is defined: this means the (user-mode) app will use ntdll functions too, and so it can call these direct ntdll-exported debug functions.

I would suggest something like this (adapted from a similar situation for the `TerminateCurrentProcess` macro at the end of this file):
```c
#if defined(_NTDDK_) || defined(NTOS_MODE_USER)
/* Kernel-Mode, or Native User-Mode */

// Here, the existing DbgPrint etc. definitions...

#elif defined(_WIN32)
/* Win32 User-Mode */

// Here, your new function and definitions...

#else
    #error "Unsupported target."
#endif
```

Reply:
OK.